tests/references/test_api.py

```diff
@@ -209,6 +209,9 @@ async def test_import_reference(
 
         task_id = reference["task"]["id"]
 
+        import_reference_task = ImportReferenceTask(client.app, task_id)
+        await import_reference_task.run()
```

Comment:
Make another separate test for the task. Keeping the API and task tests together no longer makes sense here.

Please put task test in `references/test_tasks.py` module.

Reply:
Removed this test as it is already done more extensively in `test_import_reference_task`.